### PR TITLE
[bug] Fix obvious bugs across all lectures

### LINKED ISSUE #62: [linear equations] Bug in some of the math
There is a bug whereby an equation is split into multiple equations

Screenshot 2024-09-28 at 11 59 36 am

### LINKED ISSUE #65: [networks] line breaks missing and bugs in math
There are some bugs in the math formula and the lines are not separated:

image

### LINKED ISSUE #73: [Bug] Extra Hashtag causing title errors
Titles in some of the lectures below are incorrect due to extra Hashtag:

image

### PR DESCRIPTION
**Pull request by HumphreyYang**
Hi @mmcky,

I spent some time hovering over all the lectures and fixing obvious bugs I spotted.

Please kindly merge this if it looks good.

Fixes #62 
Fixes #73 
Fixes #65

### CODE REVIEW

File: lectures/time_series_with_matrices.md [hunk 1]
```diff
@@ -366,7 +366,7 @@ $$
 以及
 
 $$
-\Sigma_y = \tilde A (\σ_u^2 I_{T \times T} ) \tilde A^T
+\Sigma_y = \tilde A (\sigma_u^2 I_{T \times T} ) \tilde A^T
```
Comment (mmcky) [inline lectures/time_series_with_matrices.md:369]: @HumphreyYang should these be unicode sigma?

**Review by mmcky — APPROVED** [review]
Nice work @HumphreyYang - thank you. I'd like to get this merged as it touches a lot of files. Does it address any of the issues (i.e. titles) -- just be nice to link them if it does. Thanks.

**Comment by mmcky** [comment]
@HumphreyYang I see the fixes lines now. Thanks.

File: lectures/time_series_with_matrices.md (hunk 1 above)
Comment (mmcky) [reply lectures/time_series_with_matrices.md:369]: I've just left this as a question.

Comment (HumphreyYang) [reply lectures/time_series_with_matrices.md:369]: Many thanks @mmcky, I think this is correct under the latex environment: )

**Pull request merged**